src/ray/protobuf/common.proto

```diff
@@ -182,6 +182,7 @@ enum ErrorType {
 message RayErrorInfo {
   oneof error {
     RayException actor_init_failure = 1;
+    ActorDeathCause actor_died_error = 2;
```

Comment:
There is already an `CreationTaskFailureContext` in `ActorDeathCause`, I think we can remove the `actor_init_failure` here. Do you think so?

Reply:
Hmm yeah good point. Let me fix that.

Reply:
This is fixed